BCH-Wunder-Project-team-4/Wunder.io
# Front page

## Link to ticket:

https://trello.com/c/ytOfddrO/65-frontpage-elements

## Link to feature environment:

[localhost](http://localhost:3000/)

## Changes proposed in this PR:

Front page style 
added 2 paragraph type Wunder story and Expert talks listing 
Expert talks listing includes content type expert story 
![Screenshot 2023-12-07 at 9 55 31](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/b1978531-639c-4910-9a41-0899011e7856)
![Screenshot 2023-12-07 at 9 56 06](https://github.com/BCH-Wunder-Project-team-4/Wunder.io/assets/117649417/35d40634-d814-47f4-bb7f-44ff6cfa6556)

## How to test:

1. commit your changes and don't forget lando drush cex 
2. merge branch and do lado drush cim 
3. Add paragraph named under story and add some text there 
4. create new content  Expert talk. fill all required fields (that should work =)) 
5. add paragraph called listing expert talks to the page 

I hope I didn't break anything =)
## Best practices:

For now Accessibility is 91 (green) but might be improved 
Performance is 63................. 
breadcrumbs works weird for now 

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
